### PR TITLE
Add soft deprecations from Foundation beta 5 on macOS

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

File: src/Foundation/EnumDesktop.cs [hunk 1]
```diff
@@ -96,6 +96,7 @@ public enum AEEventClass : uint {
 
 	// Added from NSUserNotification.h
 	[Native]
+	[Advice ("'NSUserNotification' usages should be replaced with 'UserNotifications framework'.")]
```
Comment (VincentDondain) [inline src/Foundation/EnumDesktop.cs]: You put the ' at the wrong place on this one (compared the your other advice). It's 'UserNotifications'.

**Comment by chamons** [comment]
Failed with https://gist.github.com/chamons/a0aa0467e83459062c824ea4c576cbfd

15:23:22.9212910 1) Failed : Xamarin.MTouch.MT5211
15:23:22.9213510 The output does not contain the line '.*objc-class-ref in registrar.o.*'
15:23:22.9213650 at Xamarin.Tests.Tool.AssertOutputPattern (System.String linePattern) [0x0001a] in <6a875618be704f579434f1742048f7f8>:0
15:23:22.9213750 at Xamarin.MTouch.MT5211 () [0x0008a] in <6a875618be704f579434f1742048f7f8>:0
15:23:22.9213900 

@rolfbjarne thoughts ^?

and a pretty obvious network failure.

**Comment by chamons** [comment]
build

**Comment by rolfbjarne** [comment]
@chamons MT5211 is https://github.com/xamarin/maccore/issues/897

**Comment by chamons** [comment]
@VincentDondain 👍 ?

**Pull request merged**